lib/semantics/scope.h

```diff
@@ -33,6 +37,7 @@ class Scope {
   static Scope globalScope;  // contains program-units
 
   ENUM_CLASS(Kind, System, Global, Module, MainProgram, Subprogram, DerivedType)
+  ENUM_CLASS(ImportKind, Default, Only, None, All);
```

Comment:
This could be `using ImportKind = parser::ImportStmt::Kind` but I didn't want to cause `parse-tree.h` to be included wherever `scope.h` is. Maybe there could be a shared definition in a different `.h` file.

Reply:
I've started putting a few universal enum classes like this into lib/common/fortran.h.